plugin/vim-gh-line.vim

```diff
@@ -48,14 +48,27 @@ if !exists('g:gh_cgit_url_pattern_sub')
     let g:gh_cgit_url_pattern_sub = []
 endif
 
-func! s:gh_line(action) range
+if !exists('g:git_remote')
+    let g:git_remote = ""
```

Comment:
This is a global variable applicable to all vim invocations. 
I am not sure how useful this will be given that different repositories will have different remote naming schemes.
I cannot see how this variable can be set in .vimrc.

Reply:
I've thought this through, but I still would like to keep it as global variable. This gives user the ability to set the default remote name, for example, most of the time they may only care about `origin` remote name and don't want to be bothered by the interactive input everytime when they open the same repo with multiple remotes.
Maybe this is arguable, but at least I don't see the negative side of keeping this.